docs/settings/apm-settings.asciidoc

```diff
@@ -101,4 +101,7 @@ Matcher for all source map indices. Defaults to `apm-*`.
 `xpack.apm.autoCreateApmDataView` {ess-icon}::
 Set to `false` to disable the automatic creation of the APM data view when the APM app is opened. Defaults to `true`.
 
+`xpack.apm.latestAgentVersionsFileUrl` {ess-icon}::
+Specifies the URL of a self hosted file that contains latest agent versions. Defaults to `https://apm-agent-versions.elastic.co/versions.json`.
```

Comment:
Making the url configurable would allow users in air-gapped environments to implement their own mechanism to get this information (in case of compliance of proxy policies, etc), offering similar to what we have right now in fleet.

Reply:
I highly doubt users will make the effort to build a replica of the agent version service.

Reply:
True, but at least they could disable the functionality if they don't allow network access instead of getting a failure every time a user reloads the page

Reply:
@sqren I think routing via a proxy is less unlikely?

Reply:
> True, but at least they could disable the functionality if they don't allow network access instead of getting a failure every time a user reloads the page

How can a user disable?

> @sqren I think routing via a proxy is less unlikely?

Okay, I guess that's a good escape hatch then.

Reply:
@sqren in the newest version of this file you can find this:
```
Set to `''` to disable requesting latest agent versions.
```